modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1256,6 +1258,69 @@ public void testGetObjectEntries() throws Exception {
 				"sort", "textObjectFieldName:desc"
 			).build(),
 			childObjectEntry2, childObjectEntry1);
+
+		// Aggregation facets with non-admin users
+
+		_objectEntryManager.addObjectEntry(
+			_simpleDTOConverterContext, _objectDefinition1,
+			new ObjectEntry() {
+				{
+					properties = HashMapBuilder.<String, Object>put(
+						"textObjectFieldName", "Able"
+					).build();
+				}
+			},
+			ObjectDefinitionConstants.SCOPE_COMPANY);
+
+		_user = _addUser();
+
+		Role role = RoleTestUtil.addRole(RoleConstants.TYPE_REGULAR);
+
+		_resourcePermissionLocalService.addResourcePermission(
+			_companyId, _objectDefinition1.getClassName(),
+			ResourceConstants.SCOPE_COMPANY, String.valueOf(_companyId),
+			role.getRoleId(), ActionKeys.VIEW);
+
+		_userLocalService.addRoleUser(role.getRoleId(), _user);
+
+		Aggregation aggregation = new Aggregation() {
+			{
+				setAggregationTerms(
+					HashMapBuilder.put(
+						"textObjectFieldName", "Able"
+					).build());
+			}
+		};
+
+		Page<ObjectEntry> page = _objectEntryManager.getObjectEntries(
+			_companyId, _objectDefinition1, null, aggregation,
+			new DefaultDTOConverterContext(
+				false, Collections.emptyMap(), _dtoConverterRegistry, null,
+				LocaleUtil.getDefault(), null, _user),
+			null,
+			_filterPredicateFactory.create(
+				null, _objectDefinition1.getObjectDefinitionId()),
+			null, null);
+
+		List<Facet> facets = page.getFacets();
+
+		if (ListUtil.isEmpty(facets)) {
+			Assert.fail();
+		}
+
+		Facet facet = facets.get(0);
+
+		List<Facet.FacetValue> facetValues = ListUtil.filter(
+			facet.getFacetValues(),
+			facetValue -> Objects.equals(facetValue.getTerm(), "Able"));
+
+		if (ListUtil.isEmpty(facetValues)) {
+			Assert.fail();
+		}
```

Comment:
@gabrielwas I've neve see this pattern for this case I believe assertFalse is more appropriate:

````java
Assert.assertFalse(ListUtil.isEmpty(facetValues));
````

what do you think?

Reply:
Good catch! Thx!